### DISCUSSION TITLE
Bug: stars from users that I follow do not appear in new feed

### DISCUSSION THREAD

**Discussion by cpressey** [community/community#66587] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### What I did

* Followed about 175 users, a dozen or so of whom star at least one repo every day.
* Went to Github home page today.
* Clicked "Filter" button.
* Unchecked all checkboxes.
* Checked the checkbox labelled "Stars - Repositories being starred by people".
* Clicked "Save".

### What I expected to happen

* I expected to see a list of repos that accounts I am following have starred over the past little while.

### What actually happened

* I saw only the message "That's all for now. You can adjust your feed filter to see more content, or visit Explore."

### Comments

This change to the feed is a transparent attempt to get content that I didn't ask for in front of my eyeballs.  The fact that your feed filter options speak of "people" rather than "people you follow" is a dead giveaway.  That alone will probably drive away a lot of users.

But you didn't stop there; the new feed doesn't even have _feature parity_ with the old feed!  I can't do with the new feed what I did with the old feed!  You clearly want to deprecate that way of using Github, burying it in a [hidden page](https://github.com/dashboard-feed) that doesn't even have responsive layout (and as such is nearly unusable on my phone).  This will just mean an even bigger drop in user numbers.  And, take note, _these users will be among your most perceptive and discerning users_, which are your most valuable ones.

Of course, I totally understand that, since Github is a free service, "I'm not the customer, I'm the product".  In some sense "I have no right to complain".  But I also understand that, as `git` is an open-source technology, there is basically **zero vendor lock-in** in this situation. If anyone here wanted to pull up stakes and host their repos somewhere else, it would be remarkable easy for them to do so.  In a very real sense, a large amount of Github's "trade" is not in dollars and cents, but in _credibility in the open-source sphere_.  In such "market positions" it behooves one to think very carefully about the messages that their actions send.

**Comment by sooftware** [comment]
me too.

**Comment by icebob** [comment]
Same issue, I don't see the newly starred repos of my followings.

**Reply by Ly0n** [reply]
Same issue here.

**Comment by cujomalainey** [comment]
I don't even see stars on my repos anymore

**Comment by zhzyker** [comment]
me too 
I thought it was just my account that had a problem

I really need to see my friends' stars

<img width="750" alt="Screenshot 2023-09-12 at 12 27 14" src="https://github.com/community/community/assets/32918050/e5958572-2145-4883-a650-d07d816ac0c1">

**Reply by Bian-Sh** [reply]
SAME STATUS

**Comment by Manchery** [comment]
me too

**Comment by noobpk** [comment]
Same issue here

**Comment by serpro69** [comment]
They've broken the feed entirely with their algorithmic-facepoop-like approach... Don't try to fix things that are working fine for Pete's sake...

**Comment by theAkito** [comment]
Thanks for showing the [hidden page](https://github.com/dashboard-feed), which works at least as a workaround, as long as this buggy mess of a new "feed" is broken as hell.

**Comment by theAkito** [comment]
# Workaround

Install this script. Works with Tampermonkey & Violentmonkey.

https://github.com/wangrongding/github-old-feed

**Comment by leogregianin** [comment]
Same issue, I don't see the newly starred repos of my followings.

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._


## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 


_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_


- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._


_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._

**Reply by theAkito** [reply]
> We should have done a better job communicating recent changes

Sounds to me like "you were just too stupid to understand the changes, so we need to explain them to you really slowly, then the changes will become (i.e. seem) better, trust me bro"....

I cannot believe, that an **aglorithmic** solution, requiring tons & tons of data & processing power is supposed to be more performant, than plainly & simply showing what happens with your followers.

The new solution is probably far more resource intensive.

**Comment by R3DRUN3** [comment]
Same problem for me!
Also I've noticed a great increase in the sync time of status badges (github actions).

**Comment by mohamadmansourX** [comment]
I'm encountering the same problem unable to view newly starred repositories of those I'm following.
It's quite surprising that they changed the old [dashboard feed page](https://github.com/dashboard-feed). I'm curious about the rationale behind this change, especially since the previous version seemed to be effective and widely accepted by users.

**Reply by theAkito** [reply]
They did this, because they don't want to give, what we, the users, want, but what some higher up manager at Microsoft's Github department wants. He read somewhere online, that making everything social media algorithmic is the new shit and brings you tons of money, so he ordered the lower ones to do exactly that, even though those who actually know a bit how this site is supposed to work, probably never agreed with it, internally.

**Comment by zarifpour** [comment]
To work around this, I found that if I follow 112 users or fewer, this issue doesn't occur. So, what you can do is either create a new GitHub account or modify an existing account to keep the number of users followed under this limit.

Since I mostly use the GitHub mobile app, switching between accounts is quite easy and convenient for me. I simply hold my profile in the app to switch accounts as needed. This way, I can have one account for following my preferred set of users, staying within the limit, and another account for different activities or following additional users. This workaround has been effective for me in managing the bug and ensuring that I can see the stars from the users I follow in my feed.

**Reply by ryanxingql** [reply]
I follow 89 users. But the problem still exists.

**Reply by zarifpour** [reply]
Interesting. Maybe it's different for each user? I can verify if I follow 114 users on my other account it breaks, and when I unfollow one account, to 113, it starts working.

**Comment by fasiondog** [comment]
me too. always show: 3 weeks ago

**Comment by spenserblack** [comment]
I found that I wasn't seeing stars from users I follow *until I unfollowed all organizations that I was following.* See [this post](https://github.com/orgs/community/discussions/13130#discussioncomment-7802923) for more info. If you have this issue, check if you're following any organizations, unfollow them, and see if that fixes it. It did for me.

**Reply by theAkito** [reply]
I pretty much don't follow any organisations, except maybe 2, but these follows are super recent. More recent, than this feed update.

**Reply by spenserblack** [reply]
Sorry to hear that. Perhaps [this previous comment](https://github.com/orgs/community/discussions/66587#discussioncomment-7673419) is related? Regardless, when GitHub fails to show you user stars for whatever reason, it should report that in an error alert or something instead of just making us all guess why. Is it a bug? Am I following too many users? Who knows, there's nothing telling us that :shrug:

**Reply by theAkito** [reply]
Since I switched to using the [good feed](https://github.com/wangrongding/github-old-feed), I don't really test the new one anymore. It already wasted too much of my time.

**Comment by cpressey** [comment]
Hey!  I found a solution to this bug!  And that solution was to **quit using GitHub**.  I have moved to a **non-profit code host** now, and I have found that this is **much better aligned with my values as an open-source contributor**.

Of course, when I say "quit using GitHub", I do mean "as much as humanly possible".  I'm sure you'll agree that GitHub has become "**too big to fail**".  Hopefully you can also see that there is a **significant amount of irony** in an open-source code host based on a distributed version control system becoming "too big to fail".  And when I say "significant", I mean "**unsupportable**".  So, I can no longer in good conscience support it.

Goodbye.

**Comment by ciros88** [comment]
same problem here

**Comment by PeterChen1997** [comment]
same problem......

**Comment by LiteBrite82** [comment]
💬 Your Product Feedback Has Been Submitted 🎉
Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩
- Your input will be carefully reviewed and cataloged by members of our product teams. 
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 

Where to look to see what's shipping 👀
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

What you can do in the meantime 💻
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐

**Comment by msteele999** [comment]
Still not working?

**Comment by WindZh03** [comment]
same problem

**Comment by kuantian-zhang** [comment]
same problem

**Comment by quocthinhle** [comment]
same problem lol

**Comment by zt64** [comment]
still happening here

**Comment by llussy** [comment]
same problem

**Comment by chase** [comment]
Noticed a bug where specifically I sponsor Neovim, but for some reason users who star Neovim are showing up on my feed. Seems like the logic for this is backwards.

**Comment by gibsramen** [comment]
Happening to me as well.

**Comment by LiteBrite82** [comment, marked as answer]
Hi Community!

Thank you all for your posts and comments on the recent Feed bug. The bug was related to starred repositories, which is why we see some people saying the fix is to uncheck "Include events from starred repositories".

The Feed team has pushed an update to address the issue; [please leave us feedback in this separate post](https://github.com/orgs/community/discussions/143676) so we can monitor its effectiveness.

**Comment by AmintaCCCP** [comment]
same issue +1

**Comment by doggycxj** [comment]
same issue +1

**Comment by makulski** [comment]
Same exact problem.

**Comment by Edd-G** [comment]
Confirm problem
